src/ppspp_options.erl

```diff
@@ -173,20 +183,22 @@ unpack( <<?PPSPP_LIVE_DISCARD_WINDOW, Maybe_Discard_Window/binary>>,
                  16#ffffffffffffffff -> ppspp_live_discard_window_keep_all_chunks;
                  _ -> Window0
              end,
-    Options = orddict:store(ppspp_live_discard_window, Window, Options0),
-    unpack(Maybe_Options, Options);
+    %% Options = orddict:store(ppspp_live_discard_window, Window, Options0),
+    unpack(Maybe_Options, Options#{ppspp_live_discard_window => Window});
+
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 unpack( <<?PPSPP_SUPPORTED_MESSAGES,
           Length:?BYTE,
           Maybe_Supported_Message_Types:Length/big,
-          Maybe_Options/binary >>, Options0) ->
+          Maybe_Options/binary >>, Options) ->
     Supported_Message_Types = case Maybe_Supported_Message_Types of
                                   %% TODO probably needs a custom bitmatching parser
                                   _ -> ppspp_supported_messages_not_yet_implemented
```

Comment:
Line 196 is too long:                                   _ -> ppspp_supported_messages_not_yet_implemented.